Dev

dmbavykin opened the pull request.

Commit:
hotfix

Commit:
ref

kirillshevch commented on app/views/orders/show.html.haml:
```diff
@@ -62,7 +62,7 @@
             %span.in-grey-600 Total
       %tbody
-        - @order.order_items.each do |order_item|
+        - @_order.order_items.each do |order_item|
```
@_order?

kirillshevch replied:
Add it to rubocop exceptions

dmbavykin replied:
I use ```@order ``` as instance variable of ApplicationController, so can`t use same name

Commit:
locales and new ability logic added

Commit:
more tests

Commit:
need more tests

Commit:
rubocop ref

kirillshevch commented on db/schema.rb:
```diff
@@ -10,161 +10,167 @@
 
-ActiveRecord::Schema.define(version: 20180316120551) do
-
+ActiveRecord::Schema.define(version: 20_180_326_084_545) do
```
`db/*` files should be in rubocop exceptions, because next running of `rails db:migrate` will back it again to rails style

kirillshevch replied:
add `.rubocop.yml` file with the next rules:
```yaml
AllCops:
  Exclude:
    - 'db/**/*'
  TargetRubyVersion: 2.5

Rails:
  Enabled: true
```

kirillshevch commented on .rspec:
```diff
@@ -1 +1,2 @@
 --require spec_helper
+--require rails_helper
```
enough just `--require rails_helper`, because:
https://github.com/6aBblKuH/bookstore/blob/dev/spec/rails_helper.rb#L1

kirillshevch commented on app/views/order_steps/address.html.haml:
```diff
@@ -34,4 +34,4 @@
           %td
             %p.font-16= "€#{@order.total}"
-    = submit_tag 'Save and Continue', class:'btn btn-default center-block mb-20'
+    = submit_tag 'Save and Continue', class:'btn btn-default center-block mb-20', id: 'addresses-submit-btn'
```
i18n?

Commit:
ref after review

Commit:
hotfix

	modified:   .rspec
	modified:   app/decorators/book_decorator.rb
	modified:   app/views/order_steps/address.html.haml
	modified:   app/views/order_steps/delivery.html.haml
	modified:   app/views/order_steps/payment.html.haml
	modified:   config/locales/en.yml
	modified:   config/locales/ru.yml

Commit:
set flash for invalid password updating
